models/maat/DESCRIPTION

```diff
@@ -0,0 +1,21 @@
+Package: PEcAn.MAAT
+Type: Package
+Title: PEcAn package for integration of the MAAT model
+Version: 1.4.5
+Date: 2016-03-30
+Author: Shawn Serbin, Anthony Walker
+Maintainer: Shawn Serbin <sserbin@bnl.gov>
+Description: This module provides functions to link the MAAT to PEcAn.
+Depends:
+    PEcAn.utils,
+    proto,
```

Comment:
Need to add proto to install.dependencies.R if you're using it in the package. I don't see where you're using this package, so if it's in there by mistake, take it out of the DESCRIPTION.